tensorflow/core/api_def/base_api/api_def_Greater.pbtxt

```diff
@@ -4,5 +4,15 @@ op {
   description: <<END
 *NOTE*: `Greater` supports broadcasting. More about broadcasting
 [here](http://docs.scipy.org/doc/numpy/user/basics.broadcasting.html)
+
+```python
+x = tf.constant([5, 4, 6])
+y = tf.constant([5, 2, 5])
+tf.math.greater(x,y) ==> [False, True, True]
```

Comment:
space after ,

All code examples should follow the same style as the regular code, ideally.